### PR TITLE
Create Rule “avoid-repeated-emojis/rule”

### PR DESCRIPTION
**Pull request by JackDevAU**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/avoid-repeated-emojis/rule.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Good example – using the DRY principle 
+:::
+
+
+By avoiding repeated emojis within each category, the list remains clear and concise, which can improve communication and understanding. Thus making it easy to scan the list and understand the benefits and drawbacks of a particular situation. 
+
+Following the DRY principle by avoiding excessive repetition of emojis helps to create lists that are visually interesting and easy to read, while also promoting efficient and maintainable content creation.
+
+
+::: tip
```
Comment (tiagov8) [inline rules/avoid-repeated-emojis/rule.md:61]: Hey @JackDevAU 

There is not box type "tip"
See https://ssw.com.au/rules/rule/#5-boxes 

1. Please fix
I would add just the prefix "Tip:" in bold in front of the sentence

File: rules/avoid-repeated-emojis/rule.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+When creating a list that includes emojis, avoid repeating the same emoji more than three times within the list. This helps to keep the list concise, readable, and consistent while still allowing for some repetition for emphasis or clarity.
```
Comment (tiagov8) [inline rules/avoid-repeated-emojis/rule.md:62]: 1. Please follow https://ssw.com.au/rules/use-digits-instead-of-words/

File: rules/avoid-repeated-emojis/rule.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Reduces stress
+
+### ❌ Cons
+
+- Requires effort to implement
+- May take time to adjust
+- Can be challenging to maintain
+:::
+::: good
+Good example – using the DRY principle
```
Comment (tiagov8) [inline rules/avoid-repeated-emojis/rule.md:52]: 1. Please make "Using" capitalized for consistency ;)

File: rules/avoid-repeated-emojis/rule.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    url: https://www.ssw.com.au/people/jack-pettit
+related:
+  - use-emojis
+created: 2023-02-28T00:28:10.330Z
+guid: 1acd6766-d31f-493f-b6f1-3b0b2ff2dc4e
+---
+Emojis are a lot easier to consume when a lot of information is present. But when repeated excessively, they can become a hindrance rather than a help. This helps to make it more visually interesting and easier to scan quickly. In this way, emojis can enhance the communication experience, rather than detracting from it.
+
+<!--endintro-->
+
```
Comment (tiagov8) [inline rules/avoid-repeated-emojis/rule.md:16]: Hey @JackDevAU 

1. Can you add a ::: info box here with a short defintion of DRY? E.g. https://www.digitalocean.com/community/tutorials/what-is-dry-development

> D_RY, which stands for ‘don’t repeat yourself,’ is a principle of software development that aims at reducing the repetition of patterns and code duplication in favor of abstractions and avoiding redundancy._

**Comment by tiagov8** [comment]
Hey @JackDevAU 

I was thinking... this rule can also be useful for words not just emojis... 

What do you think about changing from:

```
title: DRY - Do you avoid repeated emojis in lists?
uri: avoid-repeated-emojis
```

to

```
title: DRY - Do you avoid repeated elements in lists?
uri: avoid-repetition
```

**Note:** Require update in the category + folder name as well

---

Then add another bad and good example... e.g.:

```

  This is also valid for words in lists. You should keep only the part that is unique in each list item.

  ::: greybox
  Following this rule is important:
  
  - To help you increase productivity
  - To help you save time
  - To help you reduce stress
 
  :::
  ::: bad
  Bad example – Repeating words... Not following DRY :(
  :::

  ::: greybox
  Following this rule is important to help you:
  
  - Increase productivity
  - Save time
  - Reduce stress
 
  :::
  ::: good
  Good example – No repeated words... using the DRY principle 
  :::
```

File: rules/avoid-repeated-emojis/rule.md (hunk 3 above)
Comment (JackDevAU) [inline rules/avoid-repeated-emojis/rule.md:52]:
```suggestion
Good example – Using the DRY principle 
```

File: rules/avoid-repeated-emojis/rule.md (hunk 2 above)
Comment (JackDevAU) [inline rules/avoid-repeated-emojis/rule.md:62]:
```suggestion
When creating a list that includes emojis, avoid repeating the same emoji more than 3 times within the list. This helps to keep the list concise, readable, and consistent while still allowing for some repetition for emphasis or clarity. 
```

File: rules/avoid-repeated-emojis/rule.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+By avoiding repeated emojis within each category, the list remains clear and concise, which can improve communication and understanding. Thus making it easy to scan the list and understand the benefits and drawbacks of a particular situation. 
+
+Following the DRY principle by avoiding excessive repetition of emojis helps to create lists that are visually interesting and easy to read, while also promoting efficient and maintainable content creation.
+
+
+::: tip
+When creating a list that includes emojis, avoid repeating the same emoji more than three times within the list. This helps to keep the list concise, readable, and consistent while still allowing for some repetition for emphasis or clarity. 
+:::
```
Comment (JackDevAU) [inline rules/avoid-repeated-emojis/rule.md:63]:
```suggestion

```

File: rules/avoid-repeated-emojis/rule.md (hunk 1 above)
Comment (JackDevAU) [inline rules/avoid-repeated-emojis/rule.md:61]:
```suggestion
**Tip:**
```

**Comment by JackDevAU** [comment]
> Hey @JackDevAU
> 
> I was thinking... this rule can also be useful for words not just emojis...
> 
> What do you think about changing from:
> 
> ```
> title: DRY - Do you avoid repeated emojis in lists?
> uri: avoid-repeated-emojis
> ```
> 
> to
> 
> ```
> title: DRY - Do you avoid repeated elements in lists?
> uri: avoid-repetition
> ```
> 
> **Note:** Require update in the category + folder name as well
> 
> Then add another bad and good example... e.g.:
> 
> ```
> 
>   This is also valid for words in lists. You should keep only the part that is unique in each list item.
> 
>   ::: greybox
>   Following this rule is important:
>   
>   - To help you increase productivity
>   - To help you save time
>   - To help you reduce stress
>  
>   :::
>   ::: bad
>   Bad example – Repeating words... Not following DRY :(
>   :::
> 
>   ::: greybox
>   Following this rule is important to help you:
>   
>   - Increase productivity
>   - Save time
>   - Reduce stress
>  
>   :::
>   ::: good
>   Good example – No repeated words... using the DRY principle 
>   :::
> ```

Agreed! I've updated the title and added the example to the rule

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**